client/src/js/samples/components/__tests__/General.test.js

```diff
@@ -65,11 +59,20 @@ describe("mapStateToProps()", () => {
             isolate: "Isolate Foo",
             locale: "Bar",
             paired: false,
-            srna: false,
             gc: "31.2 %",
             count: "13.2 m",
             lengthRange: "50 - 100",
-            subtractionId: "baz"
+            subtractionId: "baz",
+            libraryType: "Normal"
         });
     });
+
+    it.each([["normal", "Normal"], ["srna", "sRNA"], ["amplicon", "Amplicon"]])(
+        "state.library_type(%s) should equal props.libraryType(%s)",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace `····"state.library_type(%s)·should·equal·props.libraryType(%s)",⏎·······` with `])("state.library_type(%s)·should·equal·props.libraryType(%s)",`](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4918216)